doc/install/index.rst

```diff
@@ -1,72 +1,74 @@
-=======================
- Installation (Manual)
-=======================
+.. _install-overview:
 
+===============
+Installing Ceph
+===============
 
-Get Software
+There are different options to install Ceph. Each solution meet different
+requirements, like ease of maintenance.
+
+
+ceph-ansible
 ============
 
-There are several methods for getting Ceph software. The easiest and most common
-method is to `get packages`_ by adding repositories for use with package
-management tools such as the Advanced Package Tool (APT) or Yellowdog Updater,
-Modified (YUM). You may also retrieve pre-compiled packages from the Ceph
-repository. Finally, you can retrieve tarballs or clone the Ceph source code
-repository and build Ceph yourself.
+Installs Ceph using Ansible.
 
+* `docs.ceph.com/ceph-ansible <https://docs.ceph.com/ceph-ansible/>`_ 
```

Comment:
Caveats:
- Ansible is not integrated with the orchestration API (new in octopus), so new management features in the CLI and dashboard will not work.

Reply:
Well, we need good way to hint users to the right tool. E.g. k8s has a table of features: https://kubernetes.io/docs/setup/#production-environment

Reply:
We need a list of all caveats of this kind.

Reply:
@sebastian-philipp This PR is now superseded by this one:https://github.com/ceph/ceph/pull/33890

If you're satisfied with this, we can close this PR.